packages/grid/_modules_/grid/hooks/features/export/seralizers/csvSeraliser.ts

```diff
@@ -35,12 +37,12 @@ interface BuildCSVOptions {
   columns: GridColumns;
   rows: Map<GridRowId, GridRowModel>;
   selectedRows: Record<string, GridRowId>;
-  getCellValue: (id: GridRowId, field: string) => GridCellValue;
+  getCellParams: (id: GridRowId, field: string) => GridCellParams;
   delimiterCharacter: GridExportCsvDelimiter;
 }
 
 export function buildCSV(options: BuildCSVOptions): string {
-  const { columns, rows, selectedRows, getCellValue, delimiterCharacter } = options;
+  const { columns, rows, selectedRows, getCellParams, delimiterCharacter } = options;
```

Comment:
Should we make it an opt out option?

Reply:
Applying the same logic then we should also make `delimiterCharacter` optional. Although you need either 'getCellParams' or `getCellValue` here to get the actual cell value because we only have access to the cell ID and using the `apiRef` in here is not ideal as that file is with helper/utility functions that are grid agnostic.
IMO we can keep it for now as it is.

Reply:
That's ok, I meant using the value formatter, there might be cases where one would prefer the raw value, maybe it should be column based as well 🤔 
What do you think? @mui-org/x

Reply:
The first step was to apply the same `valueFormatter` to the CSV export, that might be sufficient for most of the use cases. If someone requests for a per-column formatting, we can add an exclusive value formatter for the export: https://github.com/mui-org/material-ui-x/issues/1814#issuecomment-852678630

Reply:
Yes that's what we said originally. I'm just being extra cautious here about numbers, ie `1,000`, should be exported as `"1,000"`, would that be interpreted as numbers by excel? 🤔

Reply:
Just tested and it seems ok. Let's not do the option for now

Reply:
Yeah, we had a PR about that: https://github.com/mui-org/material-ui-x/pull/1656#issuecomment-840122724 If the OS is correctly configured, we just need to use the same formatting in the CSV export.